src/Button/Button.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+import { bool, func, string } from 'prop-types';
+
+import classNames from '../utils/classNames';
+
+import styles from './button.module.scss';
+
+type NrfButtonProps = {
+    id?: string;
+    className?: string;
+    onClick: () => void;
+    disabled?: boolean;
+    title?: string;
+};
+
+const NrfButton: React.FC<NrfButtonProps> = ({
+    children,
+    id,
+    className,
+    onClick,
+    disabled = false,
+    title,
+}) => (
+    <button
+        type="button"
+        id={id}
+        className={classNames(styles.button, className)}
+        disabled={disabled}
+        onClick={onClick}
+        title={title}
+    >
+        {children}
+    </button>
+);
+
+NrfButton.propTypes = {
+    id: string,
+    className: string,
+    onClick: func.isRequired,
+    disabled: bool,
+    title: string,
+};
+
+export default NrfButton;
```

Comment:
I used the `Card` component as a template here, but I agree.
I could also change Card.tsx since it is only an internal name. That way it is consistent at least.

Reply:
In [the `Card` component](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/4f0ef69cf861f22db3628da37ce0ee1a20a32841/src/Card/Card.tsx) there is a potential name clash because internally it uses the `Card` component from Bootstrap, so one of the two has to have a different name. 

It seemed easier to me to call the external component `NrfCard` instead of naming the internal one `BootstrapCard`. But maybe you have an idea for an even better solution there?

Reply:
Now that you say it. I initially used the bootstrap Button which is why I opted for the name `NrfButton`.
Changed it to `Button` now.